force-app/main/default/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -944,31 +944,32 @@ export default class GeFormRenderer extends LightningElement{
             try {
                 this.loadingText = this.CUSTOM_LABELS.geAuthorizingCreditCard;
     
-                const authorizedGift = await this.currentElevateBatch.add(tokenizedGift);
-                const isAuthorized = authorizedGift.status === this.PAYMENT_TRANSACTION_STATUS_ENUM.AUTHORIZED
-                    || authorizedGift.status === this.PAYMENT_TRANSACTION_STATUS_ENUM.PENDING;
+                const elevateBatchItem = await this.currentElevateBatch.add(tokenizedGift);
+                const isSuccessful = elevateBatchItem.status === this.PAYMENT_TRANSACTION_STATUS_ENUM.AUTHORIZED
```

Comment:
We should extract this into a function then we can call the function inline in the `if` block below